lib/src/html/html_generator_instance.dart

```diff
@@ -95,54 +97,81 @@ class HtmlGeneratorInstance implements HtmlOptions {
       generateLibrary(package, lib);
 
       for (var clazz in lib.allClasses) {
+        // TODO(jcollins-g): consider refactor so that only the canonical
+        // ModelElements show up in these lists
+        if (!clazz.isCanonical) continue;
```

Comment:
So essentially there is a always one library that a class is canonical to (not sure if I got the lingo right here) and that's the only place it should be documented? If a class is not canonical, don't document it here, as it will be documented in the canonical library.